assets/css/harper.css

```
@@ -215,6 +216,19 @@ p.byline {
     transform: translateY(0%);
 }
 
+/* Visually hidden but exposed to screen readers (list-page h1s) */
+.sr-only {
+    position: absolute;
+    width: 1px;
+    height: 1px;
+    padding: 0;
+    margin: -1px;
+    overflow: hidden;
+    clip: rect(0, 0, 0, 0);
```

Stylelint (17.14.0)

[error] Deprecated property "clip" (property-no-deprecated)

(property-no-deprecated)